content/release/ReleaseNotes3610.md

```diff
@@ -30,7 +34,7 @@ The Java team worked to **improve accuracy in Assess** for this release in these
 
 * Detecting SSRF attacks
 
-* When using Java 11.
+* When using Java 11
 
 The team also made **improvements for reporting and troubleshooting,** as well as smaller bug fixes. These include:
 
```

Comment:
A little awkward. Change to:
* Clarified usage of the max_stack_depth property and improved reporting of the error that occurs when it is misconfigured.
* Added a heartbeat message to help administrators diagnose Contrast Protect syslog connectivity.

Reply:
These are the bullet points that follow the quoted section...couldn't directly comment on those bullets, for some reason.